### PR TITLE
Fixed bug #18911 import of theme fails if it contains resource with uppercase extension

### PR DESCRIPTION
**Pull request by gabrieljenik**

### CODE REVIEW

File: application/helpers/admin/template_helper.php [hunk 1]
```diff
@@ -134,7 +134,11 @@ function templateExtractFilter($p_event, &$p_header)
     $aAllowExtensions[] = 'twig';
     $info = pathinfo($p_header['filename']);
 
-    if ($p_header['folder'] || !isset($info['extension']) || in_array($info['extension'], $aAllowExtensions)) {
+    if (isset($info['extension'])) {
+        $p_header['filename'] = preg_replace('/\.\w+$/', '.' . strtolower($info['extension']), $p_header['filename']);
```
Comment (Shnoulle) [inline application/helpers/admin/template_helper.php:138]: You rename it here ? 

I don't think it's a good idea : if user add  in is twig file

**Review by Shnoulle — COMMENTED** [review]
Dpon't rename it (if id odn't make error)

File: application/helpers/admin/template_helper.php (hunk 1 above)
Comment (gabrieljenik) [reply application/helpers/admin/template_helper.php:138]: > I don't think it's a good idea : if user add  in is twig file

He should edit it in that case.
Lower case extensions will minimize errors I believe.

@olleharstedt thoughts?

Comment (Shnoulle) [reply application/helpers/admin/template_helper.php:138]: >  Lower case extensions will minimize errors I believe.

There are no reason to didn't allow picture.PNG

Comment (gabrieljenik) [reply application/helpers/admin/template_helper.php]: Done

**Comment by olleharstedt** [comment]
Neat!

**Pull request merged**